cloudinit/config/cc_growpart.py

```diff
@@ -263,16 +263,20 @@ def device_part_info(devpath):
     bname = os.path.basename(rpath)
     syspath = "/sys/class/block/%s" % bname
 
-    # FreeBSD doesn't know of sysfs so just get everything we need from
-    # the device, like /dev/vtbd0p2.
-    if util.is_FreeBSD():
-        freebsd_part = "/dev/" + util.find_freebsd_part(devpath)
-        m = re.search("^(/dev/.+)p([0-9])$", freebsd_part)
-        return (m.group(1), m.group(2))
-    elif util.is_DragonFlyBSD():
-        dragonflybsd_part = "/dev/" + util.find_dragonflybsd_part(devpath)
-        m = re.search("^(/dev/.+)s([0-9])$", dragonflybsd_part)
-        return (m.group(1), m.group(2))
+    if util.is_BSD():
+        # FreeBSD doesn't know of sysfs so just get everything we need from
+        # the device, like /dev/vtbd0p2.
+        fpart = "/dev/" + util.find_freebsd_part(devpath)
+        m = re.search("^(/dev/.+)p([0-9])$", fpart)
+        if m:
+            return (m.group(1), m.group(2))
```

Comment:
fwiw, this is equivalent to:
```
            return m.group(1), m.group(2)
```

Reply:
I would've expected black to take care of that, like it did in the loops

Reply:
Agreed, though I guess it doesn't have an opinion on this one.

Reply:
just from looking at the other code in this file, and the codebase in general, `return (foo, bar)` seems to be way more common than `return foo, bar`

Reply:
fun fact, and irrelevant aside: FreeBSD's [`style(9)`](https://man.freebsd.org/style(9)) *requires* parenthesis for a `return` statement.

(but that has to do with C's weird precedence rules)

Reply:
> just from looking at the other code in this file, and the codebase in general, return (foo, bar) seems to be slightly more common than return foo, bar

I'm not surprised. Similarly, the codebase also used to have more tuple unpacking using syntax with unnecessary parenthesis, i.e. `(a, b) = some_func()`, which hasn't been the style trend as of late, but which is the dominant style in many files still.

Stylistically, I try to avoid using parenthesis unless they are required or unless the code is confusing without them.